components/Banner/README.md

```diff
@@ -1,8 +1,156 @@
-<!-- This file was auto-generated using ./scripts/generate_readme Banner -->
+<!-- This file was auto-generated using scripts/generate_readme Banner -->
 
 # Banner
 
 [![Open bugs badge](https://img.shields.io/badge/dynamic/json.svg?label=open%20bugs&url=https%3A%2F%2Fapi.github.com%2Fsearch%2Fissues%3Fq%3Dis%253Aopen%2Blabel%253Atype%253ABug%2Blabel%253A%255BBanner%255D&query=%24.total_count)](https://github.com/material-components/material-components-ios/issues?q=is%3Aopen+is%3Aissue+label%3Atype%3ABug+label%3A%5BBanner%5D)
 
 A banner displays a prominent message and related optional actions.
-Note: this component is currently being actively developed and is not yet ready for general use.
+
+<div class="article__asset article__asset--screenshot">
+  <img src="docs/assets/banner.png" alt="Banner" width="320">
+</div>
+
+## Design & API documentation
+
+
+## Table of contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [Installation with CocoaPods](#installation-with-cocoapods)
+  - [Importing](#importing)
+- [Usage](#usage)
+  - [Importing](#importing)
+  - [Appearance](#appearance)
+  - [Styling](#styling)
+  - [LayoutMargins](#layoutmargins)
+- [Examples](#examples)
+  - [Creating a banner view](#creating-a-banner-view)
+
+- - -
+
+## Overview
+
+`MDCBannerView` is a view that displays important, succinct message, and provides actions for users to address (or dismiss the banner). It requires a user action to be dismissed.
+
+`MDCBannerView` is currently a [beta component](../../contributing/beta_components.md). Therefore, clients that
+wish to use MDCBannerView in their app will need to manually specify the MaterialComponentsBeta target and our repository. 
+
+## Installation
+
+<!-- Extracted from docs/../../../docs/component-installation.md -->
+
+### Installation with CocoaPods
+
+Add the following to your `Podfile`:
+
+```bash
+pod 'MaterialComponents/Banner'
+```
+<!--{: .code-renderer.code-renderer--install }-->
+
+Then, run the following command:
+
+```bash
+pod install
+```
+
+### Importing
+
+To import the component:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+import MaterialComponents.MaterialBanner
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialBanner.h"
+```
+<!--</div>-->
+
+
+## Usage
+
+<!-- Extracted from docs/typical-use.md -->
+
+### Importing
```

Comment:
It looks like you are doing the Import step twice here

Reply:
Addressed in acc72d593483e098cb41c30814b599549b1eee99